demos/tab-indicator.html

```diff
@@ -0,0 +1,113 @@
+      <div class="mdc-toolbar__row">
+        <section class="mdc-toolbar__section mdc-toolbar__section--align-start">
+          <span class="catalog-back">
+            <a href="/" class="mdc-toolbar__menu-icon"><i class="material-icons">&#xE5C4;</i></a>
+          </span>
+          <span class="mdc-toolbar__title catalog-title">Tab Indicator</span>
+        </section>
+      </div>
+    </header>
+
+    <main class="mdc-toolbar-fixed-adjust">
+      <section class="hero">
+        <div class="demo-inner">
+          <div class="demo-item">
+            <label class="demo-label"><input type="radio" name="demo1" checked>&nbsp;Active</label>
+            <span class="mdc-tab-indicator mdc-tab-indicator--active demo-indicator"></span>
+          </div>
+          <div class="demo-item demo-item--wide">
+            <label class="demo-label"><input type="radio" name="demo1">&nbsp;Active</label>
+            <span class="mdc-tab-indicator demo-indicator"></span>
+          </div>
+          <div class="demo-item demo-item--narrow">
+            <label class="demo-label"><input type="radio" name="demo1">&nbsp;Active</label>
+            <span class="mdc-tab-indicator demo-indicator"></span>
+          </div>
+          <div class="demo-item">
+            <label class="demo-label"><input type="radio" name="demo1">&nbsp;Active</label>
+            <span class="mdc-tab-indicator demo-indicator"></span>
+          </div>
+        </div>
+
+        <div class="demo-inner">
+          <div class="demo-item demo-item--wide">
+            <label class="demo-label"><input type="radio" name="demo2" checked>&nbsp;Active</label>
+            <span class="mdc-tab-indicator mdc-tab-indicator--icon mdc-tab-indicator--active demo-indicator material-icons">favorite</span>
+          </div>
+          <div class="demo-item">
+            <label class="demo-label"><input type="radio" name="demo2">&nbsp;Active</label>
+            <span class="mdc-tab-indicator mdc-tab-indicator--icon demo-indicator material-icons">lens</span>
+          </div>
+          <div class="demo-item">
+            <label class="demo-label"><input type="radio" name="demo2">&nbsp;Active</label>
+            <span class="mdc-tab-indicator mdc-tab-indicator--icon demo-indicator material-icons">change_history</span>
+          </div>
+          <div class="demo-item demo-item--narrow">
+            <label class="demo-label"><input type="radio" name="demo2">&nbsp;Active</label>
+            <span class="mdc-tab-indicator mdc-tab-indicator--icon demo-indicator material-icons">star</span>
+          </div>
+        </div>
+      </section>
+    </main>
+
+    <script src="/assets/material-components-web.js" async></script>
+    <script src="/assets/common.js" async></script>
+    <script>
+      demoReady(function() {
+        var demoItems = document.querySelectorAll('.demo-item');
+        var activeIndicator;
+        for (var i = 0; i < demoItems.length; i++) {
+          (function(i) {
```

Comment:
Nit: Can you create a named function for this? An IIFE feels a little convoluted here.